security-considerations.md

```diff
@@ -11,24 +14,25 @@ First, we prove p < p(currentOrder) via contraction:
 We assume: p >= p(currentOrder)
 (In this proof all divisions are integer divisions)
 
-```
-fullAuctionAmount / prevBidSum >= buyAmountOfIter /sellAmountOfIter
-fullAuctionAmount * sellAmountOfIter >= prevBidSum * buyAmountOfIter
-fullAuctionAmount * sellAmountOfIter / buyAmountOfIter >= prevBidSum
-0 >= prevBidSum - fullAuctionAmount * sellAmountOfIter / buyAmountOfIter
-sellAmountOfIter >= prevBidSum + sellAmountOfIter - fullAuctionAmount * sellAmountOfIter / buyAmountOfIter
-sellAmountOfIter >= currentBidSum - fullAuctionAmount _ sellAmountOfIter / buyAmountOfIter
-sellAmountOfIter >= uncoveredBids
+```bin
```

Comment:
TIL you can use any string to tag code block!
Why did you pick `bin`? I don't see a clear meaning and it doesn't seem to have a custom treatment in GitHub ([list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)).